documentation/blog/2024-04-30-svelte-5-release-candidate.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Svelte 5 Release Candidate
+description: We're almost there
+author: The Svelte team
+authorURL: https://svelte.dev/
+---
+
+Svelte 5 is now in the Release Candidate phase. This means that the design of the framework is largely settled, with no anticipated breaking changes between now and the stable release, and that the most egregious bugs have been stomped.
+
+It _doesn't_ mean that it's ready for production, or that nothing will change between now and 5.0. But if you've held off on dabbling with Svelte 5 during the public beta phase, now is a great time to try it out. You can select the 'Try the Svelte 5 preview' option when starting a new SvelteKit project...
+
+```bash
+npm create svelte@latest
+```
+
+...or you can add Svelte 5 to an existing project...
+
+```bash
+npm install --save-dev svelte@next
+```
+
+...or you can tinker in the [Svelte 5 Preview](https://svelte-5-preview.vercel.app) website.
+
+## I'm out of the loop. What's Svelte 5?
+
+For the last several months, we've been hard at work rewriting Svelte from the ground up to be faster, easier to use and more robust, applying the lessons we've collectively learned from several years of building apps with Svelte.
+
+You can learn more about the new features from the [preview documentation](https://svelte-5-preview.vercel.app/docs), and by watching the presentation from the most recent [Svelte Summit](https://www.sveltesummit.com/):
+
+<div class="max">
+<figure style="max-width: 960px; margin: 0 auto">
+<div style="aspect-ratio: 1.755; position: relative; margin: 0 auto;">
+	<iframe style="position: absolute; width: 100%; height: 100%; left: 0; top: 0; margin: 0;" src="https://www.youtube-nocookie.com/embed/xCeYmdukOKI" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+
+<figcaption>What You Can Do For Your Framework</figcaption>
+</figure>
+</div>
+
+The highlights include:
+
+- [runes](/blog/runes), the new signal-powered reactivity API that sits at the heart of Svelte 5 and unlocks _universal reactivity_
+- overhauled event handling with less boilerplate and more flexibility
+- better component composition through _snippets_
```

Comment:
Do we have a nice thing to link to for snippets? Is there something friendlier than https://svelte-5-preview.vercel.app/docs/snippets ?

Reply:
not really — is that link particularly unfriendly?

Reply:
```suggestion
- better component composition through [_snippets_](https://svelte-5-preview.vercel.app/docs/snippets)
```

Reply:
Not all _that_ unfriendly, no. It's just that the first bullet point is linking to a blog post about runes - which contains a video - and not the docs page on runes. If you're fine with linking to a blog post on runes and a docs page on snippets, that works for me.